### DISCUSSION TITLE
Running in Azure without using VMs

### DISCUSSION THREAD

**Discussion by GiorgioBe** [Egbert-Azure/stockfish-cluster#3] [Ideas]
Did you consider using another type of resource (like containers) instead of virtual machines?

**Comment by Egbert-Azure** [comment]
> Did you consider using another type of resource (like containers) instead of virtual machines?

no, not yet. I have to admit my lack of knowledge for this. But might be great idea!

**Comment by GiorgioBe** [comment]
The use of containers should reduce a lot the costs in comparison to using VMs, and possibly make things even simpler.
I was wondering also about the possiiblity to use containers with GPU muscle inside, in order to use engines like Lc0.
During the holidays I'm planning to spend some time playing with this, hoping that an incoming golive won't interfere too badly with my hopes.

**Reply by Egbert-Azure** [reply]
I think I will build a home lab with 4 SBC (LePotato) and LXD and do the same using container. And within the container MPi for the workloads. So far as I understand, LXD manage the nodes and balance, while MPI is sending the messages for Stockfish running on the nodes. Again, not my core competency.
